### PR TITLE
Adding conda to get Cookiecutter

### PR DESCRIPTION
**Pull request by RichaldoElias**
[//]: # (Thank you for helping us out: your efforts mean great deal to the project and the community as a whole!)

[//]: # (Before you proceed:)

[//]: # (1. Make sure to add yourself to `CONTRIBUTORS.rst` through this PR provided you're contributing here for the first time)
[//]: # (2. Don't forget to update the `docs/` presuming others would benefit from a concise description of whatever that you're proposing)

## Description

[//]: # (What's it you're proposing?)

## Rationale

[//]: # (Why does the project need that?)

## Use case(s) / visualization(s)

[//]: # ("Better to see something once than to hear about it a thousand times.")

### CODE REVIEW

**Comment by RichaldoElias** [comment]
pip is not the only way to get Cookiecutter, we can also use  conda

File: README.rst [hunk 1]
```diff
@@ -132,8 +132,15 @@ Let's pretend you want to create a Django project called "redditclone". Rather t
 and then editing the results to include your name, email, and various configuration issues that always get forgotten until the worst possible moment, get cookiecutter_ to do all the work.
 
 First, get Cookiecutter. Trust me, it's awesome::
+There are to ways get Cookiecutter:
```
Comment (rosariopfernandes) [inline README.rst]: Did you mean two?

**Comment by browniebroke** [comment]
Correct, it's not the only way, there is also (at least):

- Homebrew
- pipx
- pipenv

Cookiecutter Django is a tempalate, I don't think we should list all of the ways to install the cookiecutter CLI, but rather document the preferred way and then link to the CLI documentation for other ways. Something along the lines of this maybe?

> First, get Cookiecutter via pip (or [via another installation method](https://cookiecutter.readthedocs.io/en/latest/installation.html#install-cookiecutter))

**Comment by browniebroke** [comment]
Thanks for the effort in trying to get this better documented, but it's not in a state to be accepted here and this is quite well documented in the CLI documentation.

**Pull request closed without merging**